plugin/evm/syncervm_test.go

```diff
@@ -426,7 +426,7 @@ type syncVMSetup struct {
 	serverAppSender *enginetest.Sender
 
 	includedAtomicTxs []*Tx
-	fundedAccounts    map[*keystore.Key]*types.StateAccount
+	fundedAccounts    map[*utils.Key]*types.StateAccount
```

Comment:
Since one of the fields in `utils.Key` is a pointer, this [doesn't guarantee uniqueness](https://go.dev/play/p/azN5EW1T39N). Is that ok?

Reply:
It seems okay as randomly generated keys would likely not be duplicate and this change does not modify this part of the behavior